clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -36,11 +36,13 @@
 #ifndef _CACHE_SIMULATOR_H_
 #define _CACHE_SIMULATOR_H_ 1
 
+#include <map>
 #include <unordered_map>
-#include "simulator.h"
+#include "../reader/config_reader.h"
```

Comment:
This should just be "config_reader.h" to avoid breaking 3rd-party integration where this header is placed in a flat directory layout

Reply:
Done.